spec/features/create_and_deliver_digests_spec.rb

```diff
@@ -0,0 +1,201 @@
+require "rails_helper"
+
+RSpec.describe "creating and delivering digests", type: :request do
+  before do
+    Timecop.freeze "2017-01-02 10:00"
+
+    allow_any_instance_of(DeliveryRequestService)
+      .to receive(:provider_name).and_return("notify")
+  end
+
+  after do
+    Timecop.return
+  end
+
+  scenario "daily digest run" do
+    login_with_internal_app
+
+    #create two subscriber lists with different links
+    list_one_topic_id = "0eb5d0f0-d384-4f27-9da8-3f9e9b22a820"
+    list_two_topic_id = "a915e039-070b-4633-813d-187af61cad7a"
+
+    stub_govdelivery("TEST123")
+    create_subscribable(title: "Subscriber list one", links: {
+      topics: [list_one_topic_id]
+    })
+
+    stub_govdelivery("TEST456")
+    create_subscribable(title: "Subscriber list two", links: {
+      topics: [list_two_topic_id]
+    })
+
+    lookup_subscribable("TEST123")
+    subscribable_one_id = JSON.parse(response.body)["subscribable"]["id"]
```

Comment:
Could we make `lookup_subscribable` return the id so this line wasn't necessary?

Reply:
:+1: I've added that.